docs/rules/test-case-shorthand-strings.md

```diff
@@ -169,6 +169,7 @@ ruleTester.run('example-rule', rule, {
 });
 ```
 
+<!-- markdownlint-disable-next-line no-duplicate-header -->
 #### `never`
```

Comment:
seems this line should be `consistent-as-needed`. :)

Reply:
Oh, nice it caught a bug!